open-company/open-company-web
# ToS and PP updates

Card: https://trello.com/c/DKKFqQlW

Updates to Terms of Service and Privacy Policy pages, ToS has only style changes, instead PP has also the reviewed copy for GDPR compliance.

Privacy Policy copy: https://docs.google.com/document/d/1pW3wDetQYN6uy1HeL4oCKlx2O1Y-LXr47toBW1bIWFc/edit#

To test:
- visit /privacy on desktop
- visit /privacy on mobile
- visit /terms on desktop
- visit /terms on mobile
- [x] all looking good? good
- now go to /privacy page
- open also the GDoc with the copy
- [x] take a random paragraph and check that it is visible and correctly reported in the page
- [x] repeat for 3 or 4 times to make sure we are not missing any copy
